services/code_hosting/git_hub_service.rb

```diff
@@ -121,13 +121,13 @@ def set_build_status!(repo: nil, sha: nil, state: nil, target_url: nil, descript
         description = "Still running" if state == "pending"
 
         # TODO: what's the difference?
-        description = "Something went wrong" if state == "failure"
-        description = "Something went wrong" if state == "error"
+        description = "Build encountered a failure" if state == "failure"
```

Comment:
Just in case we start using different states, I'd like to see it show a slightly different message